hasher-matcher-actioner/hmalib/models.py

```diff
@@ -38,6 +39,28 @@ def remove_content_key_prefix(key: str) -> str:
         return key[len(DynamoDBItem.CONTENT_KEY_PREFIX) :]
 
 
+class SNSMessage:
+    def to_sns_message(self) -> str:
+        raise NotImplementedError
+
+    @classmethod
+    def from_sns_message(cls, message: str) -> "SNSMessage":
+        raise NotImplementedError
+
+
+@dataclass
+class Label:
+    key: str
+    value: str
```

Comment:
What are example keys and values?

Reply:
eg. 
- `CallWebhook:6734023402834`
- `ApplyAction:WEBHOOK-898`
- `BankTag:cat-pictures`, `BankTag:cat-on-beach-pictures`